README.md

```diff
@@ -19,15 +32,26 @@ Run `ng generate component component-name` to generate a new component. You can
 
 Run `ng build` to build the project. The build artifacts will be stored in the `dist/` directory.
 
-## Running unit tests
+## How to run
```

Comment:
Should we change this to something like "How to run for development / debugging"
and probably add a How to run section, pointing to the docker-based section of the endpoint?